oeps/processes/oep-0058-proc-translations-management.rst

```diff
@@ -0,0 +1,169 @@
+  putting all translation files in the same place, it is easier to track translation
+  progress, and debug translation issues.
+* A repo that only contains text/binary files, and uses branches to separate translations
+  related to Open edX releases can make all interactions with translations very quick and
+  simple due to the ability to clone the branch of a specific release, with translations
+  organized by repository name.
+* By using an app that is maintained by Transifex the organization, we reduce the
+  maintenance burden and are more future proof of changes they might make since they
+  maintain both the API and the github App.
+
+Consequences
+************
+
+Impact on Translators
+=====================
+
+As we approach the end of the translation upgrade process, we will need to tactically
+move from multiple transifex projects to a single project.  This will require
+coordination with our translators to ensure that moving forward they are providing
+translations in the right place.
+
+Impact on Site Operators
+========================
+
+Currently the translation files for any given service or library is stored at the same
+place as the code, this has generally simplified the deployment story in the past.  With
+this change, the translations files will move to their own repository.  As we deprecate
+the old translations files, the relevant deployment tooling will need to be updated to
+pull down the translations from the new repo as a part of the deployment process.  This
+will impact both the old Ansible based tooling as well as any new docker based tooling.
+
+Impact on Developers
+====================
+
+While it won’t directly impact the day-to-day workflow of developers, due to the same
+reasons that we impact site operators(new translations location), we will have to update
+development tools as well.
+
+Rejected Alternatives
+*********************
+
+Rewriting the Current Tooling for the New API
+=============================================
+
+While rewriting the current tooling may solve the problems encountered in the last two
+Open edX Releases, it will still be difficult to maintain and keep track of the
+edx-transifex-bot. The Open edX community will still struggle with debugging difficult
+issues that stem from missing source code.
+
+Making a Transifex Project for each Repository
+==============================================
+
+TCRIL pays for each Transifex Project. As translation support is provided for more repos,
+the total cost will also increase. In addition, each Transifex Project will need to be
+maintained separately, making debugging issues or tracking the progress of each Transifex
+Project time-consuming.
+
+Next Steps
+**********
+
+* Move translation files to a new repo
+* Add GitHub Transifex App to openedx organization
+* Connect i18n repo to Transifex via app
+* Set up for each release branch
+* Create/Copy Transifex Memory
+
+Locations
+*********
```

Comment:
What does this section mean?

Reply:
This OEP proposes a new place of storing things, so this section elaborates on where these places will be now. This may not need its own section, and can be a subsection of an Approach/Proposed Implementation Section.

Reply:
Yea, I think move this into a Proposed Implementation section.  You could have that be a top level section and then pros and cons could go as subsections after you outline the approach.